### DISCUSSION TITLE
amamba/intro/index

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2150] [Announcements]
# amamba/intro/index

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/intro/index.html

**Comment by Catherine-monk** [comment]
安全自动化渐进式交付的图有点模糊，是否可以用清晰的图替换

**Comment by Ye-Ting** [comment]
在 DCE 5.0 中的地位 ，* 换 定位
